src/wx/widgets/keyedit.cpp

```diff
@@ -124,10 +124,12 @@ wxString wxKeyTextCtrl::ToString(int mod, int key)
 
 #endif
 
-    if (s.empty() || (key != wxT('-') && s[s.size() - 1] == wxT('-'))
-                  || (key != wxT('+') && s[s.size() - 1] == wxT('+')))
+    if (s.empty() || (key != wxT('-') && s[s.size() - 1] == wxT('-') && s != wxT("Num -"))
+                  || (key != wxT('+') && s[s.size() - 1] == wxT('+') && s != wxT("Num +")))
```

Comment:
In case you are wondering, this was breaking `WXK_NUMPAD_SUBTRACT` and `WXK_NUMPAD_SUBTRACT`   bindings we had. Check how the it was being done before and how it was translated in `keyedit.h`.